modoboa/extensions/webmail/templates/webmail/compose.html

```diff
@@ -8,7 +8,7 @@
       <div class="form-group{% if form.errors.to %} has-error{% endif %}">
         <label class="col-sm-1 control-label" for="id_to">{% trans "To" %}</label>
         <div class="col-sm-8">
-          <input id="id_to" name="to" class="form-control" value="{% if form.to.value %}{{ form.to.value }}{% endif %}"{% if form.to.errors %} placeholder="{% for error in form.to.errors %}{{ error }}{% endfor %}"{% endif %}>
+          <input id="id_to" name="to" class="form-control" value="{% if form.to.value %}{{ form.to.value }}{% endif %}"{% if form.to.errors %} placeholder="{% for error in form.to.errors %}{{ error }}{% endfor %}" title="{% for error in form.to.errors %}{{ error }}{% endfor %}"{% endif %}>
```

Comment:
Yes, I added the error message in the title, because it was set 
previously only in the placeholder field which is not displayed if the 
field is not empty.

We could do something else for the error messaging system, if you have 
an Idea...

--
vincent.

Le 03/03/2015 16:06, Antoine Nguyen a écrit :
>
> In modoboa/extensions/webmail/templates/webmail/compose.html 
> <https://github.com/tonioo/modoboa/pull/697#discussion_r25693772>:
>
> > @@ -8,7 +8,7 @@
> >        <div class="form-group{% if form.errors.to %} has-error{% endif %}">
> >          <label class="col-sm-1 control-label" for="id_to">{% trans "To" %}</label>
> >          <div class="col-sm-8">
> > -          <input id="id_to" name="to" class="form-control" value="{% if form.to.value %}{{ form.to.value }}{% endif %}"{% if form.to.errors %} placeholder="{% for error in form.to.errors %}{{ error }}{% endfor %}"{% endif %}>
> > +          <input id="id_to" name="to" class="form-control" value="{% if form.to.value %}{{ form.to.value }}{% endif %}"{% if form.to.errors %} placeholder="{% for error in form.to.errors %}{{ error }}{% endfor %}" title="{% for error in form.to.errors %}{{ error }}{% endfor %}"{% endif %}>
>
> If you include the errors into the title property, the user won't see 
> them unless he puts the mouse on the input box, right ?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/tonioo/modoboa/pull/697/files#r25693772>.
>